x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -469,17 +467,12 @@ export const update = async (
       );
     }
 
-    const updateCases: ESCasePatchRequest[] = query.cases.map((updateCase) => {
+    const updateCases: CasePatchRequest[] = query.cases.map((updateCase) => {
       const currentCase = myCases.saved_objects.find((c) => c.id === updateCase.id);
-      const { connector, ...thisCase } = updateCase;
+      const { id, version } = updateCase;
       return currentCase != null
-        ? getCaseToUpdate(currentCase.attributes, {
-            ...thisCase,
-            ...(connector != null
-              ? { connector: transformCaseConnectorToEsConnector(connector) }
```

Comment:
We no longer need to do the transformation here because that functionality/responsibility has been moved to the service layer.